en/asgardeo/docs/get-started/try-samples/ws-federation-webapp.md

```diff
@@ -0,0 +1,137 @@
+
+## Configure the service provider
+
+1. On the WSO2 Identity Server Console, go to **Applications**.
+
+2. Click **New Application** and select **Standard-Based Application**.
+
+3. Enter the following details:
+
+    ![Create a new Passice STS app]({{base_path}}/assets/img/guides/applications/create-new-ws-federation-app.png){: width="700" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+    <table>
+        <tr>
+            <td>Name</td>
+            <td>
+                Give a unique name to identify your application.
+                <p>e.g.:<code>PassiveSTSSampleApp</code></p>
+            </td>
+        </tr>
+        <tr>
+            <td>Protocol</td>
+            <td>Select <b>WS-Federation</b>.</td>
+        </tr>
+        <tr>
+            <td>Realm</td>
+            <td>
+                This should be a unique identifier for the web app. Provide the same realm name given to the web app you are configuring WS-Federation for.
+                <p>e.g.:<code>PassiveSTSSampleApp</code></p>
+            </td>
+        </tr>
+        <tr>
+            <td>Reply URL</td>
+            <td>
+                Provide the URL of the web app you are configuring WS-Federation for. This endpoint URL will handle the token response.
+                <p><code>http://localhost:8080/PassiveSTSSampleApp/index.jsp</code></p>
+            </td>
+        </tr>
+    </table>
+
+4. Click **Register** to complete the registration.
+
+5. Go to the **Protocol** section of the application, configure the following and click **Update** to save the changes.
+
+    <table>
+        <tr>
+            <td>Reply Logout URL</td>
+            <td>
+                This endpoint in your application handles the logout response from {{product_name}}.
+                <p><code>http://localhost:8080/PassiveSTSSampleApp/index.jsp</code></p>
+            </td>
+        </tr>
+    </table>
+
+6. Go to the **User Attributes** tab and click **Add User Attribute**, and add the following attributes:
+
+    - `http://wso2.org/claims/username`
+    - `http://wso2.org/claims/emailaddress`
+
+7. Select `http://wso2.org/claims/emailaddress` as the **Subject attribute**.
+
+8. Click **Update** to save your configurations.
+
+## Try it out
+
+!!! info
+    When redirecting your users to {{product_name}} Passive STS endpoint, the following (optional) parameters are sent in the request from the sample application.
+
+	- **wa=wsignin1.0**: specifies whether {{product_name}} should issue a token for the relying party (this is the default action).
+    - **wa=wsignout1.0**: specifies whether {{product_name}} should log the user out.
+    - **wreply={replyUrl}**: specifies where the response should be sent.
+
+	Using a Network tracer such as a SAML tracer is recommended to analyze the HTTP request and responses in this scenario. With a tracer, you will be able to view the parameters mentioned above and also see the SAML token that is issued from {{product_name}}.
```

Comment:
```suggestion
	Use a Network tracer such as SAML tracer to analyze the HTTP request and responses in this scenario. With a tracer, you will be able to view the parameters mentioned above and also see the SAML token that is issued from {{product_name}}.
```

Maybe we can mark this as a "note" since this is additional information.